frontend/src/components/periodical/ReferenceSummary.spec.ts

```diff
@@ -31,6 +31,8 @@ describe('ReferenceSummary', () => {
       },
     })
 
-    expect(screen.getByText('primär')).toBeInTheDocument()
+    // a plain .not.toBeInTheDocument() should be available at some time
+    // cf. https://github.com/vitest-dev/vitest/pull/6634
+    expect(() => screen.getByText('primär')).toThrowError()
```

Comment:
You could change it to
```suggestion
    expect(() => screen.queryByText('primär')).not.toBeInTheDocument()
```